### PR TITLE
chore(j-s): Queuing

### PR DESCRIPTION
**Pull request by gudjong**
# Queuing

[Áreiðanlegri vistun í Auði - nota aðra queue en Island.is](https://app.asana.com/0/1199153462262248/1202840416565185/f)

## What

- Implements simple queuing agains AWS SQS.

## Why

In preparation for more robust message handling.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/message-handler/src/main.ts [hunk 1]
```diff
@@ -3,22 +3,28 @@ import express from 'express'
...
-  const app = await NestFactory.create(AppModule)
+  const app = await NestFactory.create(AppModule, {
+    logger: LoggingModule.createLogger(),
+  })
   app.enableShutdownHooks()
-
   app.use(express.json({ type: ['application/json'] }))
 
+  // Give the app a chance to start up before starting to handle messages
+  await new Promise((resolve) => setTimeout(resolve, 10000))
```
Comment (Arnarkari93) [inline apps/judicial-system/message-handler/src/main.ts:20]: Can't this be configured somewhere else? Are there some configuration params that we can pass to the queue/runner?

File: libs/judicial-system/message/src/lib/message.service.ts [hunk 2]
```diff
@@ -1,18 +1,116 @@
...
-  postMessageToQueue(message: Message): Promise<string> {
-    return this.queueService.add(message)
+  async receiveMessageFromQueue(
+    callback: (message: Message) => Promise<boolean>,
+  ): Promise<void> {
+    return this.sqs
+      .receiveMessage({
+        QueueUrl: this.queueUrl,
+        MaxNumberOfMessages: 1,
+        WaitTimeSeconds: 10,
```
Comment (Arnarkari93) [inline libs/judicial-system/message/src/lib/message.service.ts:107]: Should this be configured with the env params?

**Review by Arnarkari93 — COMMENTED** [review]
One question

File: apps/judicial-system/message-handler/src/main.ts (hunk 1 above)
Comment (gudjong) [reply apps/judicial-system/message-handler/src/main.ts:20]: Yes, forgot to reference the config

File: libs/judicial-system/message/src/lib/message.service.ts (hunk 2 above)
Comment (gudjong) [reply libs/judicial-system/message/src/lib/message.service.ts:107]: Yes, same as before.

**Review by Arnarkari93 — APPROVED** [review]

**Pull request merged**